data/alpha/field/local-authority-nir.yaml

```diff
@@ -0,0 +1,6 @@
+text: "A local authority in Northern Ireland."
+field: "local-authority-nir"
+phase: "alpha"
+datatype: "string"
+register: ""
```

Comment:
Doesn't this have to be `register: "local-authority-nir", and the local-authority-nir register cloned into alpha?

Reply:
for the links to work, yes. but I wasn't going to get them working. it's a non-contentious register.

Reply:
@michaelabenyohai @elliecraven what's our current process?

Reply:
Sounds good to me @michaelabenyohai happy to go with option 2 as our standard practice in these cases (unless we're later inundated with support queries that this is causing problems for users)

Reply:
Thanks @elliecraven . @Jamie-Atkinson that means you will need to populate `register: "local-authority-nir` here